var/spack/repos/builtin/packages/py-jmp/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyJmp(PythonPackage):
+    """JMP is a Mixed Precision library for JAX."""
+
+    homepage = "https://github.com/deepmind/jmp"
+    url = "https://github.com/deepmind/jmp/archive/refs/tags/v0.0.2.tar.gz"
```

Comment:
Should use pypi download for this, might need to change checksum

Reply:
requirements.txt missing there, added a comment